spec/models/reports/facility_state_spec.rb

```diff
@@ -229,31 +229,32 @@
     it "counts the latest appointments scheduled per patient by days scheduled by bucket" do
-      skip "failing Feb 4th 2022 afternoon time US"
       Timecop.return do
-        facility = create(:facility)
-        patients = create_list(:patient, 2, recorded_at: 3.months.ago, assigned_facility: facility)
-        _current_month_appointments = patients.each do |patient|
-          create(:appointment, facility: facility, patient: patient, scheduled_date: 10.days.from_now, device_created_at: Time.current)
+        Timecop.freeze("#{Date.today.end_of_month} 23:00 IST") do
+          facility = create(:facility)
+          patients = create_list(:patient, 2, recorded_at: 3.months.ago, assigned_facility: facility)
+          _current_month_appointments = patients.each do |patient|
+            create(:appointment, facility: facility, patient: patient, scheduled_date: 10.days.from_now, device_created_at: Time.current)
+          end
+
+          _appointment_1_month_ago = create(:appointment,
+            facility: facility,
+            patient: patients.first,
+            scheduled_date: Date.today,
+            device_created_at: 32.days.ago)
+          _appointment_2_month_ago = create(:appointment,
+            facility: facility,
+            patient: patients.first,
+            scheduled_date: Date.today,
+            device_created_at: 63.days.ago)
+
+          RefreshReportingViews.new.refresh_v2
+
+          expect(described_class.find_by(month_date: Period.current, facility: facility).appts_scheduled_0_to_14_days).to eq 2
+          expect(described_class.find_by(month_date: Period.current, facility: facility).total_appts_scheduled).to eq 2
+          expect(described_class.find_by(month_date: Period.month(1.month.ago), facility: facility).appts_scheduled_32_to_62_days).to eq 1
+          expect(described_class.find_by(month_date: Period.month(2.month.ago), facility: facility).appts_scheduled_more_than_62_days).to eq 1
         end
-
-        _appointment_1_month_ago = create(:appointment,
-          facility: facility,
-          patient: patients.first,
-          scheduled_date: Date.today,
-          device_created_at: 32.days.ago)
-        _appointment_2_month_ago = create(:appointment,
-          facility: facility,
-          patient: patients.first,
-          scheduled_date: Date.today,
-          device_created_at: 63.days.ago)
-
-        RefreshReportingViews.new.refresh_v2
-
-        expect(described_class.find_by(month_date: Period.current, facility: facility).appts_scheduled_0_to_14_days).to eq 2
-        expect(described_class.find_by(month_date: Period.current, facility: facility).total_appts_scheduled).to eq 2
-        expect(described_class.find_by(month_date: Period.month(1.month.ago), facility: facility).appts_scheduled_32_to_62_days).to eq 1
-        expect(described_class.find_by(month_date: Period.month(2.month.ago), facility: facility).appts_scheduled_more_than_62_days).to eq 1
```

Comment:
Let's use `31.days.ago`  and  `62.days.ago`  instead of `1.month.ago` or `2.month.ago`

Reply:
Addressed in [f703355](https://github.com/simpledotorg/simple-server/pull/3375/commits/f70335578f7932b0f3311be328dc816ce5c6b198)